bpf/process/types/basic.h

```diff
@@ -2317,10 +2317,11 @@ read_call_arg(void *ctx, struct msg_generic_kprobe *e, int index, int type,
 	if (orig_off >= 16383 - min_size) {
 		return 0;
 	}
+	orig_off &= 16383;
 	args = args_off(e, orig_off);
 
 	/* Cache args offset for filter use later */
-	e->argsoff[index] = orig_off;
+	e->argsoff[index & 7] = orig_off;
```

Comment:
These types of masks always confuse me because I don't now what the actual limits are.

Is the limit here `MAX_SELECTORS`?

ref:
```
// Filter tailcalls are {kprobe,tracepoint}/{6,7,8,9,10}
// We do one tail-call per selector, so we can have up to 5 selectors.
#define MIN_FILTER_TAILCALL 6
#define MAX_FILTER_TAILCALL 10
#define MAX_SELECTORS	    (MAX_FILTER_TAILCALL - MIN_FILTER_TAILCALL + 1)
```

How about we add a 
```
#define MAX_SELECTORS_MASK 7
```

And use that for masking?

Reply:
right, macro will be better, will change